add new sample for serializer converter when using System.Text.Json (#6174)
Particular/docs.particular.net: samples/databus/file-share-databus-custom-serializer-converter/sample.md

---

> title: File Share Data Bus using converter for SystemJsonSerializer

Reviewer: ```suggestion
title: Using databus with the SystemJsonSerializer message serializer
```

I would change the title (and menu item / summary) to something like this
